jobs/JGFS_ATMOS_GEMPAK_NCDC_UPAPGIF

```diff
@@ -28,37 +29,40 @@ export COMPONENT="atmos"
 ##############################################
 # Define COM directories
 ##############################################
-export COMIN=${COMIN:-$(compath.py ${envir}/${NET}/${gfs_ver})/${RUN}.${PDY}/${cyc}/${COMPONENT}/gempak}
-export COMINgfs=${COMINgfs:-$(compath.py ${envir}/${NET}/${gfs_ver}/${RUN}.${PDY})/${cyc}/${COMPONENT}}
-export COMINobsproc=${COMINobsproc:-$(compath.py ${envir}/obsproc/${obsproc_ver})/${RUN}.${PDY}/${cyc}/${COMPONENT}}
-export COMOUT=${COMOUT:-$(compath.py -o ${NET}/${gfs_ver}/${RUN}.${PDY})/${cyc}/${COMPONENT}}
-export COMOUTwmo=${COMOUTwmo:-${COMOUT}/wmo}
+YMD=${PDY} HH=${cyc} generate_com -rx "COM_OBS"
+GRID=1p00 YMD=${PDY} HH=${cyc} generate_com -rx "COM_ATMOS_GEMPAK_1p00:COM_ATMOS_GEMPAK_TMPL"
+
+for grid in gif upper_air; do
+  gempak_dir="COM_ATMOS_GEMPAK_${grid^^}"
+  GRID=${grid} YMD=${PDY} HH=${cyc} generate_com -rx "${gempak_dir}:COM_ATMOS_GEMPAK_TMPL"
+  if [[ ! -d "${!gempak_dir}" ]]; then mkdir -m 775 -p "${!gempak_dir}"; fi
+done
+
+YMD=${PDY} HH=${cyc} generate_com -rx COM_ATMOS_WMO
+if [[ ! -d "${COM_ATMOS_WMO}" ]]; then mkdir m 775 -p "${COM_ATMOS_WMO}"; fi
```

Comment:
Missing "-". Should be:
`mkdir -m 775 -p "${COM_ATMOS_WMO}"`

Reply:
Fixed